apps/condo/domains/acquiring/schema/PaymentByLinkService.js

```diff
@@ -0,0 +1,204 @@
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const { registerMultiPaymentForOneReceipt, registerMultiPaymentForVirtualReceipt } = require('@condo/domains/acquiring/utils/serverSchema')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const { validateQRCode, BillingReceipt, BillingIntegrationOrganizationContext, BillingRecipient } = require('@condo/domains/billing/utils/serverSchema')
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const { WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    ADDRESS_IS_INVALID: {
+        mutation: 'paymentByLink',
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Provided address did not match any FIAS address',
+    },
+    BANK_ACCOUNT_IS_INVALID: {
+        mutation: 'paymentByLink',
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Provided bank account is not in the system',
+    },
+    NO_BILLING_RECEIPTS_FOUND: {
+        mutation: 'paymentByLink',
+        code: INTERNAL_ERROR,
+        type: NOT_FOUND,
+        message: 'No Billing Receipts were found for provided accounts',
+    },
+}
+
+const PaymentByLinkService = new GQLCustomSchema('PaymentByLinkService', {
+    types: [
+        {
+            access: true,
+            type: 'input PaymentByLinkInput { dv: Int!, sender: SenderFieldInput!, qrCode: String! }',
+        },
+        {
+            access: true,
+            type: 'type PaymentByLinkOutput { multiPaymentId: ID!, address: String!, unitName: String!, accountNumber: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canPaymentByLink,
+            schema: 'paymentByLink(data: PaymentByLinkInput!): PaymentByLinkOutput',
+            resolver: async (parent, args, context) => {
+                const { data: { dv, sender, qrCode } } = args
+
+                // Stage 0: validate QR code and normalize address
+                const { qrCodeFields: { PersonalAcc, BIC, PayerAddress, lastName, paymPeriod, Sum, PayeeINN, PersAcc } } = await validateQRCode(context, { dv, sender, qrCode })
+                const addressServiceClient = createInstance({ address: PayerAddress })
+                const normalizedAddress = await addressServiceClient.search(PayerAddress, { extractUnit: true })
+                
+                if (!normalizedAddress.addressKey) throw new GQLError(ERRORS.ADDRESS_IS_INVALID)
+
+                // Stage 1: find properties by addressKey
+                const properties = await Property.getAll(context, {
```

Comment:
How does it works? It allow user to pay for the properties he owns? Or it should be only particular property?